documentation/book/con-overview-distributed-tracing.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// assembly-distributed-tracing.adoc
+
+[id='con-overview-distributed-tracing-{context}']
+= Overview of distributed tracing in {ProductName}
+
+Distributed tracing allows developers and system administrators to track the progress of transactions between applications and microservices in a distributed system. This information is useful for monitoring application performance at the micro-level and investigating issues with target systems and end-user applications.
+
+In {ProductName} and data streaming platforms in general, distributed tracing facilitates the end-to-end tracking of messages: from source systems to the Kafka cluster and then to target systems and applications.
+
+As an aspect of system observability, distributed tracing complements the metrics that are available to view in xref:assembly-metrics-setup-{context}[Grafana dashboards], as well as the available loggers for each component. 
+
+.Introduction to OpenTracing and Jaeger
+
+Distributed tracing in {ProductName} is implemented using the open source {OpenTracingHome} and {JaegerHome} projects.
+
+The OpenTracing specification defines APIs that developers can use to instrument applications -- often in microservice architectures -- for distributed tracing. It is independent from the tracing system.
```

Comment:
often in microservice architectures. But not exclusively? Does that have any impact on where we say 'track the progress of transactions between applications and microservices in a distributed system'?

Reply:
Not exclusively, to my knowledge. Good point. Best to just remove this clause here and clarify what we mean in the first para:

"Distributed tracing allows developers and system administrators to track the progress of transactions between applications (and services in a microservice architecture) in a distributed system."